src/utils/roofline_calc.yaml

```diff
@@ -0,0 +1,341 @@
+---
```

Comment:
Let's discuss it later:
My expectation would be: 0400_roofline.yaml for each arch. The 0400_roofline.yaml should be designed and generalized no diff than any other yaml besides specifying cli_style/tui_style. 
Each arch only enables its supported data type. No worry about duplication across archs. Eventually, Vignesh's PR will template it with Jinja.